dsf/org.eclipse.cdt.dsf.ui/src/org/eclipse/cdt/dsf/debug/ui/actions/DsfSteppingModeTarget.java

```diff
@@ -26,23 +25,22 @@
  *
  * @since 1.0
  */
-public class DsfSteppingModeTarget implements ISteppingModeTarget, ITargetProperties {
+public class DsfSteppingModeTarget implements ISteppingModeTarget {
 
 	private static final String ID_DISASSEMBLY_VIEW = "org.eclipse.cdt.dsf.debug.ui.disassembly.view"; //$NON-NLS-1$
 
-	private final Preferences fPreferences;
+	private IEclipsePreferences fPreferences = InstanceScope.INSTANCE.getNode("org.eclipse.cdt.dsf.ui"); //$NON-NLS-1$
+	public static final String PREF_INSTRUCTION_STEPPING_MODE = "instruction_stepping_mode"; //$NON-NLS-1$
```

Comment:
You should not redefine the value here, instead use the definition in `ITargetProperties`.
Also, why is this value public?

Reply:
Moved constant to `ICDebugUIConstants`